.changes/next-release/bugfix-build-4fc7ad5.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
AFAIK Guava is only used in tests; we should remove this since it doesn't affect users of the SDK